src/foundry/client/apps/hud/chatbubble.d.ts

```diff
@@ -6,7 +6,7 @@ declare global {
    * This application displays a temporary message sent from a particular Token in the active Scene.
    * The message is displayed on the HUD layer just above the Token.
    */
-  declare class ChatBubbles {
```

Comment:
Typescript will complain it's already in an ambient context, so the `declare` keyword is superflous.